Python/Discovering Subdomains/README.md

```diff
@@ -0,0 +1,32 @@
+# <b>Discovering Subdomains</b>
+
+[![forthebadge](https://forthebadge.com/images/badges/made-with-python.svg)](https://forthebadge.com)
+
+## Discovering Subdomains : 🚀
+
+Searching for subdomains manually would take forever. Luckily, we don't have to do that, in this script, 
+I will build a subdomain scanner in Python using requests library. 
+
+## Discovering Subdomains Instructions: 👨🏻‍💻
+
+### Step 1:
+
+    Open Termnial 💻
+    
+### Step 2:
+
+   Install the required packages  that is in the `requirement.txt`
```

Comment:
Replace this by `pip install` command for doing this. Also, in python conventions, IMO `requirements.txt` is better name instead of `requirement.txt`. You can change this,